internal/services/compute/managed_disk_data_source.go

```diff
@@ -55,6 +55,54 @@ func dataSourceManagedDisk() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"encryption_settings": {
```

Comment:
settings is redundant in property names
```suggestion
			"encryption": {
```

Reply:
in snapshot and managed disk resource it is already named `encryption_settings`. Maybe let's keep it consistent with them?
https://github.com/hashicorp/terraform-provider-azurerm/blob/2443a8844acc475bff373d12ba4858b06561f126/internal/services/compute/managed_disk_resource.go#L187
https://github.com/hashicorp/terraform-provider-azurerm/blob/2443a8844acc475bff373d12ba4858b06561f126/internal/services/compute/snapshot_resource.go#L88

Reply:
👍 works for me